src/yunohost/app.py

```diff
@@ -1419,6 +1396,15 @@ def app_ssowatconf():
         redirected_urls.update(app_settings.get("redirected_urls", {}))
         redirected_regex.update(app_settings.get("redirected_regex", {}))
 
+    for domain in domains:
+        default_app = domain_config_get(domain, "feature.app.default_app")
+        if (default_app != "_none"):
+            app_settings = _get_app_settings(default_app)
```

Comment:
There are some boring edge cases to take into account, such as if restoring the domain settings *without* restoring the corresponding default app .. in which case the default app would not be installed

```suggestion
        if default_app != "_none" and _is_installed(default_app):
            app_settings = _get_app_settings(default_app)
```